packages/react-recommendations/src/useRecommendations.ts

```diff
@@ -23,15 +24,16 @@ export type UseRecommendationsProps = {
   maxRecommendations?: number;
   searchParameters?: SearchOptions;
   threshold?: number;
+  transformItems?: RecommendationTransformItems<TObject>;
```

Comment:
We can inline `RecommendationTransformItems` if we only use it here.

Reply:
Sure!

Reply:
Done in https://github.com/algolia/react-recommendations/pull/9/commits/c4e34b585b80325f2f55bef15c2d68451683c7cb